platforms/aws/giantnetes/variables.tf

```diff
@@ -265,12 +265,10 @@ variable "external_ipsec_public_ip_1" {
   default     = "95.179.153.65"
 }
 
-## generation needs to be set somewhere else
-## for now just test password
+## vpn password
 variable "external_ipsec_password" {
   description = "shared password for ipsec connection"
   type        = "string"
-  default     = "XHJYW67%45#0KJU(*56dvgyH(*$%$#TRi*G^^rxWEY"
```

Comment:
So just to make sure: This is not used anywhere atm. (i.e. no revocation/change needed)?

Reply:
Yeah, I used this when testing stuff,  but final installaion on giraffe has different password. You can easily check in installation repo by running `opsctl show secret -i terraform-secrets.yaml -k Terraform.VPNPassword`